cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_westend_config.rs

```diff
@@ -60,7 +60,7 @@ parameter_types! {
 	// see the `FEE_BOOST_PER_PARACHAIN_HEADER` constant get the meaning of this value
 	pub PriorityBoostPerParachainHeader: u64 = 1_396_340_903_540_903;
 	// see the `FEE_BOOST_PER_MESSAGE` constant to get the meaning of this value
-	pub PriorityBoostPerMessage: u64 = 182_044_444_444_444;
+	pub PriorityBoostPerMessage: u64 = 364_088_888_888_888;
```

Comment:
Seems to be double. Can you explain why we had to bump this also for 2x POV ?

Reply:
I didn't have an idea as well, so I asked the bridges guys above: https://github.com/paritytech/polkadot-sdk/pull/5884#issuecomment-2506065294